### ISSUE TITLE
Grunt install fails

### ISSUE THREAD

**Issue by davidmehren** [sharelatex/sharelatex#456]
Hi, I just tried to build the docker container myself, but the execution  of `grunt install` fails:

```
[...]
Finished installing contacts.
Warning: Task "dirs" not found. Use --force to continue.

Aborted due to warnings.
```

I believe the cause of this is line 88 of Gruntfile.coffee where `install:dirs` is added to the `install:all` task.

**Comment by SveLil** [comment]
The task was deleted with commit 51aca345d524ceaa53023c41fb62df0fae654d23 but the function that it called was also deleted from Helper
Using release branch worked for me

**Comment by davidmehren** [comment]
@SveLil I'm sorry, I can't follow you. The `install:all` target still calls `install:dirs`: https://github.com/sharelatex/sharelatex/blob/master/Gruntfile.coffee#L88

Anyway, switching the `git clone` in the Dockerfile to clone the `release` branch fixed the problem.

**Comment by henryoswald** [comment]
That should be fixed with 5b0a6eb384b7cd1918c2c40ab60f7005f1054893
